### PR TITLE
[DEVOPS-531] Handle separate distro configs

### PR DESCRIPTION
**Pull request by Jasper-Ben**
Handle distro config split for deploy and maintenance firmware in
Makefile and CI

### CODE REVIEW

**Comment by Jasper-Ben** [comment]
merge after https://github.com/iris-GmbH/meta-iris-base/pull/125

**Comment by Eximmius** [comment]
https://wiki.iris-sensing.net/x/xQF1Fw does something needs to be updated here afterwards?

**Comment by Jasper-Ben** [comment]
> https://wiki.iris-sensing.net/x/xQF1Fw does something needs to be updated here afterwards?

yes. However, merging is currently still blocked by introduction of gitlab runner builds anyways. Will update afterwards.

**Comment by Jasper-Ben** [comment]
it would actually be smarter to use the distro configurations as new base config and statically include the configuration currently in the base config. This would remove an unnecessary, but currently mandatory dynamic include.

Reminder to myself to fix this.

**Review by martin31821 — APPROVED** [review]

File: kas-irma6-base-common.yml [hunk 1]
```diff
@@ -29,7 +27,7 @@ repos:
   # iris-specific repos
   meta-iris-base:
     url: "https://github.com/iris-GmbH/meta-iris-base.git"
-    refspec: "develop"
+    refspec: "feature/jaor/deploy_maintenance_distro_configs"
```
Comment (DanielBaumgart) [inline kas-irma6-base-common.yml:30]: Vor dem Merge wieder auf "develop" stellen!

**Review by DanielBaumgart — CHANGES_REQUESTED** [review]

File: kas-irma6-base-common.yml (hunk 1 above)
Comment (Jasper-Ben) [reply kas-irma6-base-common.yml]: Done

**Review by DanielBaumgart — APPROVED** [review]

**Comment by Jasper-Ben** [comment]
Merged into https://github.com/iris-GmbH/iris-kas/pull/112

**Pull request closed without merging**